Formula/elasticsearch@6.rb

```diff
@@ -1,8 +1,8 @@
 class ElasticsearchAT6 < Formula
   desc "Distributed search & analytics engine"
   homepage "https://www.elastic.co/products/elasticsearch"
-  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-6.8.11.tar.gz"
-  sha256 "2d136743072ca810f1af982f1c5a270b9fcc970be923ca1ee967aa7b9836cd3e"
+  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-6.8.12.tar.gz"
+  sha256 "feb6c43fe66055360754597350c088025b40566cee16175b005e55660d9e62fd"
   license "Apache-2.0"
   revision 1
```

Comment:
```suggestion
```

The revision has to be dropped when bumping the package version.

Reply:
thanks, missed that